src/utils/email_providers.py

```
@@ -0,0 +1,144 @@
+    'orange.fr',
+    'laposte.net',
+    'libero.it',
+    'virgilio.it',
+    'tiscali.it',
+    'uol.com.br',
+    'bol.com.br',
+    'terra.com.br',
+    'ig.com.br',
+    'me.com',
+    'mac.com',
+    'icloud.com',
```

Ruff (0.14.10)

Sets should not contain duplicate item `'icloud.com'`

Remove duplicate item

(B033)